Extension/test/integrationTests/IntelliSenseFeatures/index.ts

```diff
@@ -1,42 +1,38 @@
-/* --------------------------------------------------------------------------------------------
- * Copyright (c) Microsoft Corporation. All Rights Reserved.
- * See 'LICENSE' in the project root for license information.
- * ------------------------------------------------------------------------------------------ */
-
-import * as fs from "fs";
 import * as path from 'path';
-
-import { subscribeToAllLoggers } from "../../../src/logger";
-
-//
-// PLEASE DO NOT MODIFY / DELETE UNLESS YOU KNOW WHAT YOU ARE DOING
-//
-// This file is providing the test runner to use when running extension tests.
-// By default the test runner in use is Mocha based.
-//
-// You can provide your own test runner if you want to override it by exporting
-// a function run(testRoot: string, clb: (error:Error) => void) that the extension
-// host can call to run the tests. The test runner is expected to use console.log
-// to report the results back to the caller. When the tests are finished, return
-// a possible error to the callback or null if none.
-
-import testRunner = require('vscode/lib/testrunner');
-
-// You can directly control Mocha options by uncommenting the following lines
-// See https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options for more info
-testRunner.configure({
-    ui: 'tdd', 		// the TDD UI is being used in *.test.ts (suite, test, etc.)
-    useColors: true, // colored output from test results
-    fullStackTrace: true,
-    timeout: 120000
-});
-
-const logFolder: string = path.join(__dirname, ".logs");
-
-if (!fs.existsSync(logFolder)) {
-    fs.mkdirSync(logFolder);
-}
-const logFilePath: string = path.join(logFolder, "intelliSenseFeaturesTests.log");
-subscribeToAllLoggers(message => fs.appendFileSync(logFilePath, message));
-
-module.exports = testRunner;
+import * as Mocha from 'mocha';
+import * as glob from 'glob';
+
+export function run(): Promise<void> {
+  // Create the mocha test
+  const mocha = new Mocha({
+    ui: 'tdd'
+  });
+  mocha.useColors(true);
+
+  const testsRoot = path.resolve(__dirname, '.');
+
+  return new Promise((c, e) => {
+    glob('**/**.test.js', { cwd: testsRoot }, (err, files) => {

```

Comment:
nit: There seems to be inconsistent spacing in the tests.  I'm seeing 2-space and tabbed files.  Can you normalize them all to use 4-spaces?

Reply:
FYI, the test runner (new index.ts) and runTest.ts were boiler-plate files provided by the vscode-test migration documentation.  I could change them, but there would also be value in keeping them as similar to the boiler-plate files as possible.